lib/spotlight/dor/indexer.rb

```diff
@@ -40,67 +24,126 @@ class Indexer < GDor::Indexer
     before_index :add_series
     before_index :mods_cartographics_indexing
 
-    def solr_client
-      @solr_client
-    end
+    attr_reader :solr_client
```

Comment:
@cbeer   so this wasn't overriding gdor-indexer?  guess not if indexing works right.